src/react/contexts/chat-client-context.tsx

```diff
@@ -0,0 +1,73 @@
+import { ChatClient } from '@ably/chat';
+import React from 'react';
+
+// Just as in the Ably React context, we need to make sure we never create more than one Ably Chat React context.
+// This might happen when exporting a context directly from a module -
+// there's a risk of creating multiple instances of the same context
+// if there are misconfigurations in module bundler or package manager on the consumer side of Ably Chat Context.
+// This can lead to problems like having a Chat Client instance added
```

Comment:
This comment appears to be floating - can we associate it with a line of code, or delete it?